internal/provider/provider.go

```diff
@@ -106,43 +106,44 @@ func Provider() *schema.Provider {
-			"tfe_terraform_version":             resourceTFETerraformVersion(),
-			"tfe_workspace":                     resourceTFEWorkspace(),
-			"tfe_workspace_run_task":            resourceTFEWorkspaceRunTask(),
-			"tfe_variable_set":                  resourceTFEVariableSet(),
-			"tfe_workspace_variable_set":        resourceTFEWorkspaceVariableSet(),
-			"tfe_workspace_policy_set":          resourceTFEWorkspacePolicySet(),
-			"tfe_workspace_run":                 resourceTFEWorkspaceRun(),
+			"tfe_admin_organization_settings":    resourceTFEAdminOrganizationSettings(),
+			"tfe_agent_pool":                     resourceTFEAgentPool(),
+			"tfe_agent_pool_allowed_workspaces":  resourceTFEAgentPoolAllowedWorkspaces(),
+			"tfe_agent_token":                    resourceTFEAgentToken(),
+			"tfe_notification_configuration":     resourceTFENotificationConfiguration(),
+			"tfe_oauth_client":                   resourceTFEOAuthClient(),
+			"tfe_organization":                   resourceTFEOrganization(),
+			"tfe_organization_membership":        resourceTFEOrganizationMembership(),
+			"tfe_organization_module_sharing":    resourceTFEOrganizationModuleSharing(),
+			"tfe_organization_run_task":          resourceTFEOrganizationRunTask(),
+			"tfe_organization_token":             resourceTFEOrganizationToken(),
+			"tfe_policy":                         resourceTFEPolicy(),
+			"tfe_policy_set":                     resourceTFEPolicySet(),
+			"tfe_policy_set_parameter":           resourceTFEPolicySetParameter(),
+			"tfe_project":                        resourceTFEProject(),
+			"tfe_project_policy_set":             resourceTFEProjectPolicySet(),
+			"tfe_project_variable_set":           resourceTFEProjectVariableSet(),
+			"tfe_registry_module":                resourceTFERegistryModule(),
+			"tfe_no_code_module":                 resourceTFENoCodeModule(),
+			"tfe_run_trigger":                    resourceTFERunTrigger(),
+			"tfe_sentinel_policy":                resourceTFESentinelPolicy(),
+			"tfe_ssh_key":                        resourceTFESSHKey(),
+			"tfe_team":                           resourceTFETeam(),
+			"tfe_team_access":                    resourceTFETeamAccess(),
+			"tfe_team_organization_member":       resourceTFETeamOrganizationMember(),
+			"tfe_team_organization_members":      resourceTFETeamOrganizationMembers(),
+			"tfe_team_project_access":            resourceTFETeamProjectAccess(),
+			"tfe_team_member":                    resourceTFETeamMember(),
+			"tfe_team_members":                   resourceTFETeamMembers(),
+			"tfe_team_token":                     resourceTFETeamToken(),
+			"tfe_terraform_version":              resourceTFETerraformVersion(),
+			"tfe_workspace":                      resourceTFEWorkspace(),
+			"tfe_workspace_run_task":             resourceTFEWorkspaceRunTask(),
+			"tfe_variable_set":                   resourceTFEVariableSet(),
+			"tfe_workspace_variable_set":         resourceTFEWorkspaceVariableSet(),
```

Comment:
@Netra2104 I just noticed this line while looking for your resource. Can we move `"tfe_workspace_variable_set"` down to the bottom, I believe the intention here is to keep the resources alphabetized?